.github/workflows/pages.yml

```diff
@@ -0,0 +1,82 @@
+# Workflow for building and deploying a the /docs folder as static Jekyll site to GitHub Pages
+name: Deploy Jekyll site to Pages
+
+on:
+  push:
+    branches:
+      - "main"
+      - "markdown-documentation"
```

Comment:
Remember that when you're going to merge this PR, this branch is going to be deleted, so you can probably remove it now.

Reply:
Also, remove it from the allowed branches in the environment protection rules.

Reply:
Done